translator/translate/otel/processor/jmxprocessor/translator.go

```diff
@@ -0,0 +1,106 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package jmxprocessor
```

Comment:
Can you include `testdata` that has a JSON with all of this configured? I want to see what this will look like when it's being used.

Reply:
Sure, I will have the json and yaml for testing